flake.nix

```diff
@@ -13,29 +13,55 @@
         system.aarch64-darwin
       ];
 
-      ghcVersion = "9.2.4";
+      ghc = "ghc924";
 
     in flake-utils.lib.eachSystem systems (system:
       let
-        overlays = [ ];
-        pkgs = import nixpkgs { inherit system overlays; };
+        pkgs = import nixpkgs { 
+          inherit system; 
 
-        haskellPackages =
-          # Mangle the version string into the proper attribute name, then use that.
-          let attr = with pkgs.lib; "ghc" + (concatStrings (splitString "." ghcVersion));
-          in pkgs.haskell.packages."${attr}";
+          config.packageOverrides = import (nix/overlays.nix) {
```

Comment:
```suggestion
          overlays = import (nix/overlays.nix) {
```

Reply:
I will open a separate PR for this fixing overlays and CI.